File: Photon Tools/Servers/Game/python/scenes/Map_1/NPCs/Object 1/Guardian.py
from MathematicsHelper import *
from InterestManagement import *
from Game.Application.GameObjects import *

import os
import json

path = os.path.abspath(__file__)
jsonPath = os.path.splitext(path)[0] + ".json"

with open(jsonPath) as data_file:
    data = json.load(data_file)

name = data["Name"]
x = data["Position"]["x"]
y = data["Position"]["y"]
direction = data["Direction"]

position = Vector2(x, y)

message = "Dangerously there!!"

if direction > 0:
	guardian = Guardian(name, position, Direction.Left)
	scene.CreateSceneObject(guardian)
	guardian.CreateBubbleMessageNotifier(message)
else:
	guardian = Guardian(name, position, Direction.Right)
	scene.CreateSceneObject(guardian)
	guardian.CreateBubbleMessageNotifier(message)

File: Maple Fighters/MainServers/Game/Game/Application/Game Objects/NPCs/Guardian.cs
using InterestManagement;
using MathematicsHelper;

namespace Game.Application.GameObjects
{
    public class Guardian : GameObject
    {
        public Guardian(string name, Vector2 position, Direction direction) 
            : base(name, new TransformDetails(position, Vector2.Zero, direction))
        {
            // Left blank intentionally
        }
    }
}